contributing-guides/maintainers-guide.md

```diff
@@ -56,35 +56,45 @@ as a guideline for current and future maintainers.
 
 - PRs should be merged once they
   (1) **pass the automated tests** (GitHub Actions, CLA signing, etc.),
-  (2) have the **review comments addressed**,
-  (3) get **approved reviews by two maintainers**, (the second maintainer can merge immediately after approving) and
-  (4) have been open for at least **24 hours** unless the changes are trivial
-
-- If a PR fails to get a review from a second maintainer after a few days,
-  the first maintainer should ping others for review. If it still lingers around
+  (2) have all the **review comments addressed**,
+  (3) get **approved reviews by at least two maintainers**, (the second maintainer can merge immediately after approving) and
+  (4) have been open for at least **12 hours** unless the changes are trivial (contains only typo fixes or shell builtin/keyword/quoting syntax fixes)
+  
+  These rules mean that trivial PRs can be merged before 12 hours left.
+
+- If a PR fails to get a review from a second maintainer after a week,
+  the first maintainer should ping others for review. If it still lingers
   for **over a week without a second maintainer’s approval**,
-  the first maintainer can go ahead and merge it.
-
-- If the only issues holding up a merge are **trivial fixes**
-  (typos, syntax errors, etc.), and the author doesn't respond in a day or two,
-  **maintainers can make the necessary changes themselves**,
-  and proceed with the merge process.
+  the first maintainer can go ahead and merge it while possibly (if he/she/they) wants dismissing all reviewers who
+  didn't reviewed the PR.
 
 - If a PR **stops getting feedback from the submitter** and is marked as stale
   by [probot-stale](../.github/workflows/stale.yml),
   any maintainer can choose to take over the PR
   and make the necessary changes to get the content ready for merging.
 
 - During the review process, make sure that contributors, especially new ones,
-  are not **overwhelmed with too many change requests**.
+  are not **overwhelmed with too many change requests** (you being a maintainer can
```

Comment:
Not currently happy with this wording, maybe someone else can help fix it because it isn't very clear IMO

Reply:
Can you tell me what exactly u don't like in the current wording?

Reply:
I think the current wording is fine. Not sure what to change in it.